### DISCUSSION TITLE
stabilityDays not working as expected?

### DISCUSSION THREAD

**Discussion by rprovodenko** [renovatebot/renovate#14378] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

We're using npm and have `stabilityDays` set to 3. How I would imagine it working: renovate installs package versions that are older than 3 days old. How it really works: renovate checks that the packages it installed are at least 3 days old. If a new package version is released, it installs that and waits again. So if you have a large monorepo it's mostly impossible to merge the renovate PR because renovate keeps updating the PR. Is this feature supposed to work for non-trivial repos?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
When you created this bug report, did you see this prominent text?

![image](https://user-images.githubusercontent.com/6311784/155396438-530ca325-da70-4e2b-9d93-0c597eda54bd.png)

I have converted your question to a discussion this time, but please don't make the same mistake again.

**Comment by rarkins** [comment, marked as answer]
See https://docs.renovatebot.com/configuration-options/#internalchecksfilter
